Extensions/dist/extensions.js

```diff
@@ -157,3 +167,11 @@ function writeListFile(extensions) {
   });
   writePageFile('list.json', JSON.stringify(list));
 }
+
+if (require.main === module) {
```

Comment:
Exactly. I will update create-extensions-dist to call this instead of invoking it directly, which will in turn simplify this quite a bit. I need to get groceries now, but will be back soon.

Thank you for teaching me more about gulp and node modules in a day than I have learned in a year!